.circleci/config.yml

```diff
@@ -46,16 +46,17 @@ jobs:
       go_version:
         type: string
     environment:
-      CI_E2E_FILENAME: "fa6ad40d/rel-nightly"
+      CI_E2E_FILENAME: "f9acfc35/rel-nightly"
     steps:
       - go/install:
           version: << parameters.go_version >>
       - install_dependencies
       - install_linter
       - run_tests
       # Change this to run_e2e_tests once we have stable algod binaries containing delta APIs
-      # - run_e2e_tests_nightly
+      - run_e2e_tests_nightly
```

Comment:
`run_e2e_tests` is still failing

Reply:
Do you have the output from the failure? I only ask because the only differences here should be that we grab the algod binary from the stable channel instead of nightly, and all of the changes we need for them to work should be in stable.

Reply:
failed to retrieve `StateDelta`. Here's the failing test on a draft PR: https://app.circleci.com/pipelines/github/algorand/indexer/3802/workflows/e15022f5-a6a5-4911-95a6-19f03949d029/jobs/5345

Reply:
I believe it's a bug. There is no deltas for round 0. We shouldn't be making that request, https://github.com/algorand/conduit/blob/master/conduit/plugins/importers/algod/algod_importer.go#L234

Reply:
Thanks for pointing out the bug @shiqizng. I have a PR out to fix it: https://github.com/algorand/conduit/pull/78

Unfortunately, that was a red herring. We can see that after I linked my E2E indexer test branch to the conduit branch with the fix, I still fail the test but this time with even less information:
```sh
indexer-e2e-1     | ERROR:e2e_indexer.validate_accounting:failed to fetch 'http://localhost:9890/v2/accounts/BAKLOSILWYXN2ZYNP77KIF42LWTO3XM6SIXJEVRMQB64PFBGG3AGF2OAVM?round=31'
```

When I run it locally verbose, the first detailed **ERROR:** I get is the following:

```sh
indexer-e2e-1     | ERROR:e2e_indexer.validate_accounting:failed to fetch 'http://localhost:9890/v2/accounts/BAKLOSILWYXN2ZYNP77KIF42LWTO3XM6SIXJEVRMQB64PFBGG3AGF2OAVM?round=31'
```

Reply:
I believe the original comment here still applies so that once the current `rel-nightly' is released the test will pass.